src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -54,6 +54,7 @@
     private static final String TEST_REPO_DELTA = "https://github.com/reposense/testrepo-Delta.git";
     private static final String TEST_REPO_MINIMAL_STANDALONE_CONFIG =
             "https://github.com/reposense/testrepo-minimalstandaloneconfig.git";
+    private static final String INVALID_TEST_REPO = "ftp://github.com/reposense/testrepo-Delta.git";
```

Comment:
should be TEST_REPO_INVALID_LOCATION

Reply:
Thanks for the suggestion. I have changed it accordingly